tests/unit/Hooks/ImageHooksTest.php

```diff
@@ -112,7 +117,7 @@ public function testPostCreate() {
 			->method('insert')
 			->with($this->imageDimension);
 
-		$this->imagehooks->post_create($jpgFile);
+		$this->mockImageHooks->postCreate($jpgFile);
```

Comment:
Ah, sorry, I didn't understand that. I didn't notice you were talking about `ImageHooks` I'm not getting a segfault here, so maybe a PHP update went bad or a module was recently updated. The logs should tell you what went wrong.